React/Views/RCTViewManager.m

```diff
@@ -369,8 +369,14 @@ - (RCTShadowView *)shadowView
 // macOS properties
 RCT_CUSTOM_VIEW_PROPERTY(acceptsKeyboardFocus, BOOL, RCTView)
 {
-  if ([view respondsToSelector:@selector(setAcceptsKeyboardFocus:)]) {
-    view.acceptsKeyboardFocus = json ? [RCTConvert BOOL:json] : defaultView.acceptsKeyboardFocus;
+  if ([view respondsToSelector:@selector(setFocusable:)]) {
+    view.focusable = json ? [RCTConvert BOOL:json] : defaultView.focusable;
```

Comment:
Does this mean that we still support the deprecated prop, but using it results in triggering the new native stuff?

Reply:
The native stuff is pretty much the same apart from some renaming. We map both native view properties focusable and acceptsKeyboardFocus to the same code to set focusability.